SCHEMA_CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ The following document describes the changes to the JSON schema that
 as a changelog for the code in the `mir-json` tools themselves, which are
 versioned separately.)
 
+## 8
+
+Add new `InlineType` case for `TyKind::Coroutine`.
```

Comment:
Strictly speaking, this case isn't new, right? Rather, an existing case for `Coroutine` has been updated to include additional information.

Reply:
Fixed in 82c4eef4